Documentation/planning/arcade-powered-source-build/img/implementation-plan-graph.dgml

```diff
@@ -8,8 +8,8 @@
     <Node Id="aspnet-xdt" Category="Stage1" Label="aspnet-xdt" />
     <Node Id="aspnetcore" Category="Stage1" Label="aspnetcore" />
     <Node Id="aspnetcore-tooling" Category="Stage1" Label="aspnetcore-tooling" />
-    <Node Id="cecil" Category="Stage0" Label="cecil" />
```

Comment:
Do we not need cecil in SB 5.0?

Reply:
That's the plan, apparently. See https://github.com/dotnet/source-build/issues/1537#issuecomment-698282742